source/includes/steps-fle-convert-to-a-remote-master-key-aws.yaml

```diff
@@ -61,7 +97,7 @@ content: |
         :widths: 30 15 45
 
         * - Field
-          - Required
+          - Required for IAM User
```

Comment:
[suggestion] I think this table is a little confusing because whether a field is required is noted differently for users and roles.

Suggest adding a column for "Required for IAM Role" instead of this one "Required" column and the sentence "If you authenticate with an IAM role, you must specify this field." 

Field | Required for IAM Users | Required for IAM Roles | Description
x.      | Yes                                   | Yes
y.      | Yes                                   | Yes
z.      | No                                     | Yes

Reply:
Nice suggestion! I think this is cleaner.